dvc.yaml

```diff
@@ -0,0 +1,52 @@
+stages:
+  get_data:
+    cmd: python src/data/get_data.py
+    deps:
+    - src/data/get_data.py
+    outs:
+    - output/raw_x_train.joblib
+    - output/raw_y_train.joblib
+    - output/raw_x_test.joblib
+    - output/raw_y_test.joblib
+    - output/raw_x_val.joblib
+    - output/raw_y_val.joblib
+  preprocess:
+    cmd: python src/features/process_data.py
+    deps:
+    - src/features/process_data.py
+    - output/raw_x_train.joblib
+    - output/raw_y_train.joblib
+    - output/raw_x_test.joblib
+    - output/raw_y_test.joblib
+    - output/raw_x_val.joblib
+    - output/raw_y_val.joblib
+    outs:
+    - output/char_index.joblib
+    - output/x_train.joblib
+    - output/y_train.joblib
+    - output/x_test.joblib
+    - output/y_test.joblib
+    - output/x_val.joblib
+    - output/y_val.joblib
+  define_model:
+    cmd: python src/models/model_definition.py
+    deps:
+    - src/models/model_definition.py
+    - output/char_index.joblib
+  train:
+    cmd: python src/models/train.py
+    deps:
+    - src/models/train.py
+    - output/x_train.joblib
+    - output/y_train.joblib
+    - output/x_val.joblib
+    - output/y_val.joblib
+    outs:
+    - output/model.joblib
```

Comment:
Since model_definition is just a utility file, it doesnt have a main method to run, I don't think it makes sense to have it as a seperate part of the pipeline, rather just add it as a dep to the train part

Reply:
Good point, merged it into the train part